MSAL/src/public/configuration/MSALPublicClientApplicationConfig.h

```diff
@@ -110,7 +116,22 @@ to target MSAL at a specific test slice & flight. These apply to all requests ma
  */
 - (nonnull instancetype)initWithClientId:(NSString *)clientId
                              redirectUri:(nullable NSString *)redirectUri
-                               authority:(nullable MSALAuthority *)authority NS_DESIGNATED_INITIALIZER;
+                               authority:(nullable MSALAuthority *)authority DEPRECATED_MSG_ATTRIBUTE("Use initWithClientId:redirectUri:authority:nestedAuthBrokerClientId:nestedAuthBrokerRedirectUri instead");
```

Comment:
We shouldn't be deprecating this and forcing everyone to pass nested uris

Reply:
Makes sense, updated.